docs/docs/sql/decimal.md

```diff
@@ -0,0 +1,83 @@
+---
+id: sql-decimal
+sidebar_position: 7
```

Comment:
this has to maintained manually?

Reply:
indeed, there is another mode where you don't have to assign the numbers but you define the order in a file. I was thinking ti's better to switch to that mode. Right now you have to go change the numbers in all files every time you change something.